### ISSUE TITLE
Geforce Experiance

### ISSUE THREAD

**Issue by byW3iSS** [Flowseal/zapret-discord-youtube#546]
Не работает Nvidea Share и Geforce Experiance вместе с Zapret

**Comment by Flowseal** [comment]
Я лично пользуюсь Nvidia App, можеште попробовать его

**Comment by byW3iSS** [comment]
Даже с Nvidia App возникают проблемы в работе оверлея

**Comment by serpen7** [comment]
> Даже с Nvidia App возникают проблемы в работе оверлея

Это известные баги что экспириенса что нвидиа апп в их последних релизах, тут запрет ни при чем. Просто нвидиа скатилась. Запрет никак не взаимодействует с этими программами. Инста реплеи так же поломаны уже месяца 2 если не больше.

**Comment by byW3iSS** [comment]
Почему тогда при его отключении откаты и тд работает?

**Comment by serpen7** [comment]
> Почему тогда при его отключении откаты и тд работает?

потому что оно работает но не всегда, баги они такие не постоянные ведь правда? Вы говорите об софте для сетевого мониторинга трафика и его модификации который законфигурирован на правилах и листах с конфигом, который по вашим словам влияет на приложение локальное и его функции, когда они даже не прописаны в нем и уж тем более не перехватывают другие приложения, а только трафик ибо это фильтр трафика и его модификации. При этом вы приписываете это вообще в взаимодействии с утилитой на которую это не влияет и не должно влиять впринципе. Тем более на ее функциональность **ЛОКАЛЬНО**. Эффект плацебо? 

Вопрос единственно что может быть к вашему конфигу, но его не предоставлено. Поэтому не имею понятия. Может у вас там вообще конфиг чуть ли не на весь траффик, или старый конфиг с портами фиг пойми откуда. Или сборка откуда-то не пойми откуда - я не знаю. Могу лишь предполагать и знать что утилиты нвидии и без запрета поломаны уже как несколько месяцев. Ибо я этим тоже пользовался и удалил нахер. Мне стабильность в ПК дороже. А то что они понабрали к себе хрен пойми кого и кто делает там хрен пойми что - вам покажет форум нвидии, что вы не одни и вас не пару сотен или тысяч. У некоторых вообще БСОДЫ после установки дров через GFE, сносят и ставят отдельным пакетом и вдруг все нормально. Магия ведь правда?

При стандартном конфиге даже с портами udp=50000-50100 которые не пользует нвидиа и тем более уж на других портах трафика куда тоже прикручен лист хостов с которых фильтровать у вас **НЕ ДОЛЖНО** влиять ни на что от нвидии. Потому что банально там используются другие порты и банально хосты не прописаны в листах чтобы программа фильтровала даже их трафик.  Банально прога взаимодействует только с фильтруемым трафиком на который настроена на ПК в целом, не с программами.  Поэтому все конфликты с программами кроме античитов буквально высосаны из пальца. Буквально "програма имеет баги, имеет нестабильность, делает хрен пойми что в последних обновах, но я запускал недавно это и значит это виновато", ну как это так, где логика? Античиты не любят проги мониторинга трафика, его перехвата и модификации еще с давних времен.

**Comment by Eptiy** [comment]
Похожая проблема была на другом билде: https://github.com/N3M1X10/zapret_build/issues/25.

**Comment by byW3iSS** [comment]
Да, проблема именно в службе. А не как говорит ответчики выше, билд запрета у меня самый обычный 1.5.1 от автора использовал General(alt) но с другими конфигами от автора в том числе не работало, делал чистый запуск и все работает без проблем без службы, как только включается служба так и появляется данная проблема.

**Comment by serpen7** [comment]
> Да, проблема именно в службе. А не как говорит ответчики выше, билд запрета у меня самый обычный 1.5.1 от автора использовал General(alt) но с другими конфигами от автора в том числе не работало, делал чистый запуск и все работает без проблем без службы, как только включается служба так и появляется данная проблема.

кстати да а че мы фигней то страдаем у джифорса же есть логгирование, вы можете воспроизвести ошибку и посмотреть логи вот тут %USERPROFILE%\AppData\Local\NVIDIA Corporation\NVIDIA GeForce Experience ?
